src/webui/www/private/shareratio.html

```diff
@@ -14,16 +14,23 @@
         var UseGlobalLimit = -2;
         var NoLimit = -1;
 
-        var keyboardEvents = new Keyboard({
+        new Keyboard({
             defaultEventType: 'keydown',
             events: {
                 'enter': function(event) {
                     $('save').click();
                     event.preventDefault();
+                },
+                'Escape': function(event) {
```

Comment:
I don't think so because of the 'enter' event varying between instances

Reply:
I don't know if OOP approach is applicable here (in JavaScript)... I would inherit Keyboard class so that it has these fields set to some default value and then just set unique fields in its instances.

Reply:
I don't think we can achieve that with our version of JavaScript.

Reply:
>I don't think we can achieve that with our version of JavaScript.

offtopic now, perhaps we should setup webpack/babel? which can translate new ECMAScript 2015+ code to older versions.